### PR TITLE
docs: update `require-unicode-regexp.md` as following up #17402

### PR DESCRIPTION
**Pull request by sosukesuzuki**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Address review comments in https://github.com/eslint/eslint/pull/17402

### CODE REVIEW

**Comment by sosukesuzuki** [comment]
A workflow has been failed but it isn't related to this PR changes. The token at https://github.com/eslint/eslint/blob/main/.github/workflows/add-to-triage.yml#L19 seems to be invalid?

**Comment by sosukesuzuki** [comment]
The same workflow in https://github.com/eslint/eslint/pull/17439 is passed. So maybe the secret can't be used in PRs from people who don't have permission to write to the repository...?

**Comment by mdjermanovic** [comment]
> The same workflow in #17439 is passed. So maybe the secret can't be used in PRs from people who don't have permission to write to the repository...?

Yes, that seems to be the problem. We're working on the fix, isn't a blocker for merging this PR.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks! Leaving open for @nzakas to verify if his review comments from https://github.com/eslint/eslint/pull/17402 are addressed.

File: docs/src/rules/require-unicode-regexp.md [hunk 1]
```diff
@@ -99,4 +100,4 @@ function f(flags) {
 
 ## When Not To Use It
 
-If you don't want to notify regular expressions with no `u` flag, then it's safe to disable this rule.
+If you don't want to notify regular expressions with no `u` and `v` flag, then it's safe to disable this rule.
```
Comment (nzakas) [inline docs/src/rules/require-unicode-regexp.md:103]:
```suggestion
If you don't want to warn on regular expressions without either a `u` or a `v` flag, then it's safe to disable this rule.
```

**Review by nzakas — APPROVED** [review]
I cleaned up a bit of the language. LGTM.

**Pull request merged**